javascript/node/selenium-webdriver/chrome.js

```diff
@@ -346,7 +209,7 @@ class Options extends chromium.Options {
   }
 
   /**
-   * Sets the directory to store Chrome minidumps in. This option is only
+   * Sets the directory to store Chrome minidump in. This option is only
```

Comment:
`minidumps` is the correct term

Reply:
Thank you for reviewing @AutomatedTester. I have reverted it.